core/src/test/scala/tech/beshu/ror/unit/acl/blocks/rules/auth/JwtAuthRuleTests.scala

```diff
@@ -65,7 +65,7 @@ class JwtAuthRuleTests
             AuthorizationTokenDef(Header.Name.authorization, "Bearer "),
             SignatureCheckMethod.Hmac(secret.getEncoded),
             userClaim = None,
-            groupsClaim = None
+            groupsConfig = None
```

Comment:
do we cover all cases we have converted in the LDAP case?
for sure it would be nice to check the case where `ids.size != names.size`

Reply:
I added a few tests with group names and improved some test assertions.